libmscore/cmd.cpp

```diff
@@ -1565,11 +1565,13 @@ void Score::upDown(bool up, UpDownMode mode)
             if ((oNote->pitch() != newPitch) || (oNote->tpc1() != newTpc1) || oNote->tpc2() != newTpc2) {
                   // remove accidental if present to make sure
                   // user added accidentals are removed here.
-                  auto l = oNote->linkList();
-                  for (ScoreElement* e : l) {
-                        Note* ln = toNote(e);
-                        if (ln->accidental())
-                              undo(new RemoveElement(ln->accidental()));
+                  if (mode != UpDownMode::OCTAVE) {
```

Comment:
Could you please add a short comment to the code on why do we still want to preserve user accidentals in `OCTAVE` mode? It is not clear from the code right now why this check is added here.

Reply:
Done.